### ISSUE TITLE
The first Check Status gave old info on the phone.

### ISSUE THREAD

**Issue by jcomeauictx** [MyTurn-OpenSource/pyturn#1]
using BLU phone (reported by @DavidStodolsky )

**Comment by jcomeauictx** [comment]
steps to duplicate please? I just tested with 2 phones and don't see it. @DavidStodolsky

**Comment by DavidStodolsky** [comment]
This applies to the old interface with the "check status" button - http://uwsgi-alpha.myturn.mobi
After the speaker name was updated in Mac Chrome, a "check status" on the phone still showed the old speaker name. The next "check status" on the phone updated the speaker.

**Comment by jcomeauictx** [comment]
passing the test at https://github.com/MyTurn-OpenSource/pyturn/blob/alpha/apptest.py line 202 shows this *shouldn't* happen. I'm not saying it didn't, but until I can duplicate it I can't fix it. marking it closed unless new info comes in.

**Comment by jcomeauictx** [comment]
reopening, just saw it happen:

```
FAIL: test_issue_1 (__main__.TestMyturnMultiUser)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "apptest.py", line 226, in test_issue_1
    self.assertEqual('alice', status.split()[-1])
AssertionError: 'alice' != 'charlie'
- alice
+ charlie
```

**Comment by jcomeauictx** [comment]
it was an intermittent bug in the test itself, not in the server or client code. fixed now, as of commit 982c018.
